### PR TITLE
feature(eti): add groups test

### PR DESCRIPTION
**Pull request by ovsyanik-ka**
По техническим причинам создала повторный ПР

https://jira.railsc.ru/browse/AT-26

### CODE REVIEW

File: lib/apress/selenium_eti/spec/company_site/eti/product_groups_spec.rb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    log_in_as(:user)
+    navigate_to_eti
+    @cs_main_page.close_banner
+  end
+
+  describe 'Добавление групп' do
+    context 'когда привязываем группу к товару' do
+      before(:all) do
+        @product = {
+            name: Faker::Number.number(5)
```
Comment (apress-hound) [inline lib/apress/selenium_eti/spec/company_site/eti/product_groups_spec.rb]: Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.

**Comment by ovsyanik-ka** [comment]
@rezerbit

File: lib/apress/selenium_eti/spec/company_site/eti/product_groups_spec.rb [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+    log_in_as(:user)
+    navigate_to_eti
+    @cs_main_page.close_banner
+  end
+
+  describe 'Добавление групп' do
+    context 'когда привязываем группу к товару' do
+      before(:all) do
+        @product = {
```
Comment (rezerbit) [inline lib/apress/selenium_eti/spec/company_site/eti/product_groups_spec.rb]: `@product = { name: Faker::Number.number(5) }`

File: lib/apress/selenium_eti/spec/company_site/eti/product_groups_spec.rb [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+    log_in_as(:user)
+    navigate_to_eti
+    @cs_main_page.close_banner
+  end
+
+  describe 'Добавление групп' do
+    context 'когда привязываем группу к товару' do
+      before(:all) do
+        @product = { name: Faker::Number.number(5) }
```
Comment (apress-hound) [inline lib/apress/selenium_eti/spec/company_site/eti/product_groups_spec.rb]: Space inside { detected.<br>Space inside } detected.

Comment (apress-hound) [inline lib/apress/selenium_eti/spec/company_site/eti/product_groups_spec.rb]: Space inside { detected.<br>Space inside } detected.

**Pull request merged**